sessions/src/main/kotlin/io/spine/examples/pingh/sessions/RemoteGitHubAuthentication.kt

```diff
@@ -80,9 +80,9 @@ public class RemoteGitHubAuthentication(
      * similar to a successful request. Therefore, the content of the response body
      * should also be checked to ensure the request was successful.
      *
-     * @param deviceCode the code used to verify the device.
-     * @throws CannotObtainAccessToken if GitHub returns an error response
-     *                                 when trying to get an access token.
+     * @param deviceCode The code used to verify the device.
+     * @throws CannotObtainAccessToken If GitHub returns an error response
```

Comment:
No, the guidelines say nothing about `@throws`, which were read perfect sentences exactly as they were.

Please review throughout the changeset.